### ISSUE TITLE
telegram-cli not working on android-5

### ISSUE THREAD

**Issue by zorro** [termux/termux-packages#4645]
**Problem description**
The command telegram-cli gets Killed immediately. No error messages. No warnings. Just shows Killed

**Steps to reproduce**
Install telegram-cli on a android-5 device and try to run the command

**Expected behavior**
telegram-cli must start without any error

**Additional information**
termux-info:Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://termux.net stable main
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-21 science stable
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-21 games stable
Updatable packages:
All packages up to date
Android version:
5.1
Kernel build information:
Linux localhost 3.10.65 #1 SMP PREEMPT Wed Jun 29 12:03:32 CST 2016 armv7l Android
Device manufacturer:
itel
Device model:
itel it1508

Strace Log:
9603  execve("/data/data/com.termux/files/usr/bin/telegram-cli", ["telegram-cli"], ["SHELL=/data/data/com.termux/files/usr/bin/bash", "PREFIX=/data/data/com.termux/files/usr", "PWD=/data/data/com.termux/files/home", "EXTERNAL_STORAGE=/storage/sdcard0", "LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so", "HOME=/data/data/com.termux/files/home", "LANG=en_US.UTF-8", "TMPDIR=/data/data/com.termux/files/usr/tmp", "ANDROID_DATA=/data", "TERM=xterm-256color", "SHLVL=1", "ANDROID_ROOT=/system", "LD_LIBRARY_PATH=/data/data/com.termux/files/usr/lib", "PATH=/data/data/com.termux/files/home/.yarn/bin:/data/data/com.termux/files/home/.config/yarn/global/node_modules/.bin:/data/data/com.termux/files/usr/bin:/data/data/com.termux/files/usr/bin/applets", "_=/data/data/com.termux/files/usr/bin/strace"]) = ?
9603  +++ killed by SIGKILL +++

**Comment by xeffyr** [comment]
Package was moved from unstable-repo too early.
I'll remove it since it won't receive any fix anyway (really it should be rewritten from scratch).

**Comment by Talmio216** [comment]
Telegram-cli seems to get `Killed` on any armv7l device, not depending on Android version. I've tried to compile it on Android 5, 6 and 7, both in Termux and in different chrooted Linux distros (Ubuntu, Debian 9, 10 and 11, Arch, Kali) in Linux Deploy. Tested on Lenovo Tab 2 A7-30DC and Samsung Galaxy Tab 2. 
It also didn't start on a device with i686 architecture (Asus K01A, tried both in Termux and Linux Deploy), although I've successfully compiled and started it on an i386 computer with Linux Mint Tricia.
